code/modules/surgery/generic.dm

```diff
@@ -21,6 +21,90 @@
 			return 0
 		return 1
 
+/datum/surgery_step/generic/cut_with_laser
+	allowed_tools = list(
+	/obj/item/weapon/scalpel/laser3 = 95, \
+	/obj/item/weapon/scalpel/laser2 = 85, \
+	/obj/item/weapon/scalpel/laser1 = 75, \
+	/obj/item/weapon/melee/energy/sword = 5
+	)
+
+	min_duration = 90
+	max_duration = 110
+
+	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		if(..())
+			var/obj/item/organ/external/affected = target.get_organ(target_zone)
+			return affected.open == 0 && target_zone != "mouth"
+
+	begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		var/obj/item/organ/external/affected = target.get_organ(target_zone)
+		user.visible_message("[user] starts the bloodless incision on [target]'s [affected.name] with \the [tool].", \
+		"You start the bloodless incision on [target]'s [affected.name] with \the [tool].")
+		target.custom_pain("You feel a horrible, searing pain in your [affected.name]!",1)
+		..()
+
+	end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		var/obj/item/organ/external/affected = target.get_organ(target_zone)
+		user.visible_message("\blue [user] has made a bloodless incision on [target]'s [affected.name] with \the [tool].", \
```

Comment:
They make me cry, too, but I don't want to fix all of bay's shitty surgery text macro code either.

Yes. Lazy.